repository/repository.go

```diff
@@ -6,29 +6,41 @@ import (
 	"io/ioutil"
 	"path"
 	"strings"
+	"time"
 
 	logger "github.com/sirupsen/logrus"
 )
 
 type Blockchain struct {
-	ID           string   `json:"_id"`
-	BlockchainID string   `json:"blockchain"`
-	ChainID      string   `json:"chainID"`
-	Aliases      []string `json:"blockchainAliases"`
+	ID                string   `json:"id"`
+	Altruist          string   `json:"altruist"`
+	Blockchain        string   `json:"blockchain"`
+	BlockchainAliases []string `json:"blockchainAliase"`
+	ChainID           string   `json:"chainID`
+	ChaindIDCheck     string   `json:"chainIDCheck"`
+	Description       string   `json:"description"`
+	Index             int64    `json:"index"`
+	LogLimitBlocks    int64    `json:"logLimitBlocks"`
+	Network           string   `json:"network"`
+	NetworkID         string   `json:"networkID"`
+	NodeCount         int64    `json:"nodeCount"`
```

Comment:
Does `NodeCount` need to actually be stored? I thought it would be a calculated field.